gui/wxpython/mapswipe/frame.py

```diff
@@ -267,72 +253,76 @@ def OnAutoRenderChanged(self, value):
         style ^= wx.SP_LIVE_UPDATE
         self.splitter.SetWindowStyle(style)
 
-    def AddToolbars(self):
+    def AddToolbar(self, name):
```

Comment:
Not sure why not keep AddToolbars as it was. The ifs there don't make much sense, also passing the toolbar names in constructor is not particularly useful unless you create different mapswipe tools with different toolbars.

Reply:
Well, I just wanted to unify names and functionality since I call AddToolbar in ShowAllToolbars method in the base class.

Reply:
It is true that we do not need ShowAllToolbars method for Map Swipe but everything which in the base class could be generally usable in all childs. So, if we let AddToolbars the ShowAllToolbars method will need to override.

Reply:
Ok, that makes sense, so feel free to put it back. Perhaps the names of toolbars don't need to be in the constructor since they are fixed anyway,

Reply:
Also for that case when AddToolbar would be used more times (while adding/removing), then you need to adjust the code to not create a new toolbar every time.